### PR TITLE
Upgrade `explorer-schema` to Octane

### PR DESCRIPTION
**Pull request by janzenisaac**

### CODE REVIEW

File: assets/javascripts/discourse/controllers/admin-plugins-explorer.js [hunk 1]
```diff
@@ -175,10 +175,8 @@ export default Controller.extend({
   },
 
   actions: {
-    dummy() {},
-
-    expandSchema() {
-      this.set("hideSchema", false);
+    updateHideSchema(value) {
+      this.set("hideSchema", value);
```
Comment (CvX) [inline assets/javascripts/discourse/controllers/admin-plugins-explorer.js:179]: Is `hideSchema` used anywhere still in the controller/its template?

File: assets/javascripts/discourse/components/explorer-schema.js [hunk 2]
```diff
@@ -48,21 +47,24 @@ export default Component.extend({
...
 
-  @discourseComputed("filter")
-  rfilter(filter) {
-    if (!isBlank(filter)) {
-      return new RegExp(filter);
+  get filteredTables() {
+    let tables = [];
+    let filter = this.filter;
+
+    const schema = this.transformedSchema;
```
Comment (CvX) [inline assets/javascripts/discourse/components/explorer-schema.js:56]: It comes down to personal preference but I usually don't create local vars from `this` props. Makes the code easier to understand w/o context. E.g. seeing `schema` on line 67/68 I'm thinking "what's that?" I scroll up to see it's just `this.transformedSchema`

File: assets/javascripts/discourse/controllers/admin-plugins-explorer.js (hunk 1 above)
Comment (janzenisaac) [reply assets/javascripts/discourse/controllers/admin-plugins-explorer.js:179]: It is used the in `admin-plugins-explorer` template!

File: assets/javascripts/discourse/components/explorer-schema.js (hunk 2 above)
Comment (janzenisaac) [reply assets/javascripts/discourse/components/explorer-schema.js:56]: In this case we have to have a variable assignment for `this.filter` as we are manipulating the value without wanting to update the tracked value, but I agree and will update the `schema` const :)

**Review by CvX — APPROVED** [review]

**Pull request merged**